openmmtools/testsystems.py

```diff
@@ -3473,6 +3474,82 @@ def __init__(self, constraints=app.HBonds, rigid_water=True, nonbondedCutoff=DEF
 
         self.system, self.positions = system, positions
 
+# =============================================================================================
+# Drew-Dickerson B-DNA dodecamer in explicit solvent
+# =============================================================================================
+
+class DDDExplicit(TestSystem):
```

Comment:
How about we call it `DNADodecamerExplicit` or `DrewDickersonDNADodecamerExplicit`?